File: pycrayfish/api/datatypes/datapoints/data/CrayfishInputData.py
class CrayfishInputData:
    def __init__(self, datapointBatch):
        self.datapointBatch = datapointBatch

    def get(self):
        return self.datapointBatch

    def toString(self):
        return "DataBatch{datapointBatch=" + self.datapointBatch + "}"

File: core/src/main/java/datatypes/datapoints/CrayfishInputData.java
package datatypes.datapoints;

import java.io.Serializable;
import java.util.ArrayList;

public class CrayfishInputData implements Serializable {
    private ArrayList<ArrayList<Float>> datapointBatch;

    public CrayfishInputData(ArrayList<ArrayList<Float>> datapointBatch) {
        this.datapointBatch = datapointBatch;
    }

    public ArrayList<ArrayList<Float>> get() {
        return datapointBatch;
    }

    @Override
    public String toString() {
        return "DataBatch{" +
               "datapointBatch=" + datapointBatch +
               '}';
    }
}
